tests/resources/expected/test_reorder_function_decls/transformation_1.cpp

```diff
@@ -0,0 +1,26 @@
+void foo1(int n) ;
+
+void foo2(int n) ;
+
+void foo3(int n) ;
+
+int main() {
+    foo1(1);
+    foo2(2);
+    foo3(3);
+    return 0;
+}
+
+void foo3(int n) {
+    foo1(3);
+    foo2(4);
+}
+
+void foo2(int n) {
+    foo1(2);
+}
+
+void foo1(int n) {
+    int x = 9;
+}
```

Comment:
In test case, you sort functions by name, right? But here is the reverse order, why?